specifyweb/frontend/js_src/lib/components/WorkBench/hooks.ts

```diff
@@ -442,59 +443,56 @@ export function getHotHooks(wbView: WbView) {
       ];
 
       sortConfigIsSet = true;
-      getHotPlugin(wbView.hot, 'multiColumnSorting').sort(fullSortConfig);
+      getHotPlugin(workbench.hot, 'multiColumnSorting').sort(fullSortConfig);
       sortConfigIsSet = false;
 
       return false;
     },
 
     // Cache sort config to preserve column sort order across sessions
     afterColumnSort: async (_previousSortConfig, sortConfig) => {
-      if (wbView.hot === undefined) return;
+      if (workbench.hot === undefined) return;
       const physicalSortConfig = sortConfig.map((rest) => ({
         ...rest,
-        physicalCol: wbView.hot!.toPhysicalColumn(rest.column),
+        physicalCol: workbench.hot!.toPhysicalColumn(rest.column),
       }));
       setCache(
         'workBenchSortConfig',
-        `${schema.domainLevelIds.collection}_${wbView.dataset.id}`,
+        `${schema.domainLevelIds.collection}_${workbench.dataset.id}`,
         physicalSortConfig
       );
     },
 
     beforeColumnMove: (_columnIndexes, _finalIndex, dropIndex) =>
       // Don't allow moving columns when isReadOnly
-      !wbView.uploadedView &&
-      !wbView.coordinateConverterView &&
+      !isReadOnly &&
```

Comment:
You're right, the hook is preventing the columns from moving after upload. Will change this to only block moving when upload results are open. Do we need to block it when coordinates converter dialog is open? I don't see why we needed that in the original code.

Reply:
I am not sure if there could be any potential adverse side effects with allowing the columns to be moved when the Coordinate Converter dialog is open, but if it would be easy to do so I can think of one concrete reason to disallow the columns being moved when the Coordinate Converter Dialog is opened. 

The Coordinate Converter Dialog is non-modal, so any modification elements behind the dialog should probably be ignored, as they were likely unintentional. 

The only problem I can imagine with disabling column moves is if Handsontable is re-rendered while the Coordinate Converter Dialog is open: in which case the columns will once again lose their user-defined visual order until the dialog is closed. 
Would this be likely?

Reply:
Personally I think blocking column move when the dialog is open is over-engineering. I don't think most users are gonna change something in the background accidentally. Even if they do, they always have the revert button to fall back on.

I have blocked column moves when results view is open.

Reply:
> Personally I think blocking column move when the dialog is open is over-engineering

this might have been needed back when we were using jquery dialogs rather than react dialogs
jquery dialogs were conflicting with handsontable (handsontable could sometimes be partially rendered on top of jquery dialogs, and even if modal jquery dialogs were open, handsontable would still intercept some interactions)